src/ontology/edits/oeo-social.omn

```diff
@@ -277,8 +271,10 @@ Class: OEO_00000107
 Class: OEO_00000128
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A demand sector is a general term for different consumptions (households; industry; commercial sector; transport). It includes only energy demand sectors."@en,
-        rdfs:label "demand sector"
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An energy demand sector is a sector that includes energy consumers."@en,
```

Comment:
Better "covers" - includes may imply that there are also other elements included..